apps/chat/src/utils/app/id.ts

```diff
@@ -28,6 +28,9 @@ export const getConversationRootId = (bucket?: string) =>
 export const getPromptRootId = (bucket?: string) =>
   getRootId({ featureType: FeatureType.Prompt, bucket });
 
+export const getFileRootId = (bucket?: string) =>
```

Comment:
`getRootId` by default uses `featureType: FeatureType.File`. Are you sure that we need this separate method?

Reply:
I think it would be better to come to one view, either use getRootId everywhere (remove getConversationRootId and getPromptRootId methods) and pass featureType or make methods (in this case add getFileRootId) to make it more readable. For example, calling getRootId() without parameters, I don't know what id to expect from it.